docs/developer-guide/plugins.md

```diff
@@ -240,6 +240,7 @@ It accepts an options object and a callback that is invoked with warnings from t
 - `ruleSettings`: settings for the rule you are invoking
 - `root`: the root node to run this rule against
 - `result?`: the PostCSS result for resolving and invoking custom rules
+- `context?`: the PluginContext for the rule you are invoking
```

Comment:
[suggest] Since `PluginContext` is an internal name, could you change it like this?

Also, what about adding a link to the document that explains the `context` object?

```suggestion
- `context?`: the [context](rules.md#add-autofix) for the rule you are invoking
```

Reply:
Great suggestion! Definitely an improvement to the information architecture and developer experience.